webapp/publisher/snaps/build_views.py

```diff
@@ -151,6 +192,10 @@ def validate_repo(github_token, snap_name, gh_owner, gh_repo):
     return result
 
 
+def get_build_log(log_url):
```

Comment:
I'd anticipate this isn't the right place to do this, or the right thing to do....

Reply:
Hey Luke, Instead of doing this here, why not modify the `webapp/api/launchpad.py` with a method that directly return the log content?